src/Files.App/Views/BaseShellPage.cs

```diff
@@ -284,19 +285,10 @@ protected void ShellPage_PreviewKeyDown(object sender, KeyRoutedEventArgs args)
 				// Ctrl + ` (accent key), open terminal
 				case (true, _, false, true, (VirtualKey)192):
```

Comment:
Isn't this supposed to be removed?

Reply:
> We can't bind that directly to a `KeyboardAccelerator` since there's no `VirtualKey` for `` ` ``

Reply:
@cinqmilleans any solutions for this?

Reply:
I could not test because my keyboard does not have a direct ` key. If you use this shortcut, whole countries will not have access to it.

I'm not sure what the problem is you are having. You should not use KeyboardAccelerator yourself. KeyboardAccelerators are handled automatically at MainPage. Just give a HotKey for it to be used. It is also displayed in context menus automatically using ContextFlyoutItemViewModelBuilder. To display the Hotkey in another menu, you can use the HotKeyText property of IRichCommand. It does not display anything when there is no HotKey.

It would also be necessary to modify the parser and tostring of the struct HotKey to use the string `. It will look prettier in the menus.

Reply:
I realize that there are a few subtleties to be aware of. I will write a short guide to implementing actions and using commands. We will put this in the documentation.

Reply:
> I could not test because my keyboard does not have a direct ` key. If you use this shortcut, whole countries will not have access to it.

You don't need to press specifically `` ` `` to use that shortcut. Windows localize \` by default, for example I can use `ò` to trigger this shortcut

Reply:
@yaira2 this is the same way it was implemented in #11502 see file `src/Files.App/Views/LayoutModes/DetailsLayoutBrowser.xaml.cs`